Assets/HoloToolkit/Input/Scripts/Cursor/Cursor.cs

```diff
@@ -0,0 +1,478 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using UnityEngine;
+
+namespace HoloToolkit.Unity.InputModule
+{
+    /// <summary>
+    /// Object that represents a cursor in 3D space controlled by gaze.
+    /// </summary>
+    public abstract class Cursor : MonoBehaviour, ICursor
+    {
+        /// <summary>
+        /// Enum for current cursor state
+        /// </summary>
+        public enum CursorStateEnum
+        {
+            /// <summary>
+            /// Useful for releasing external override.
+            /// See <c>CursorStateEnum.Contextual</c>
+            /// </summary>
+            None = -1,
+            /// <summary>
+            /// Not IsHandVisible
+            /// </summary>
+            Observe, 
+            /// <summary>
+            /// Not IsHandVisible AND not IsInputSourceDown AND TargetedObject exists
+            /// </summary>
+            ObserveHover, 
+            /// <summary>
+            /// IsHandVisible AND not IsInputSourceDown AND TargetedObject is NULL
+            /// </summary>
+            Interact,
```

Comment:
Feedback is a gift :) 
If Ready is nebulous then the OOBE experience should not be using that verbiage.
Anyway I can see pushback on this. So please consider this non-blocking.